### PR TITLE
CCO-21: followup - adding links to STS content in a few key places

### PR DESCRIPTION
**Pull request by jeana-redhat**
Adding some links to AWS install content to make STS option easier to find.

Previews:
- [Requirements for installing OpenShift Container Platform on AWS](https://deploy-preview-34224--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/preparing-to-install-on-aws.html#requirements-for-installing-ocp-on-aws)
- [Alternatives to storing administrator-level secrets in the kube-system project](https://deploy-preview-34224--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/manually-creating-iam.html#alternatives-to-storing-admin-secrets-in-kube-system_manually-creating-iam-aws)
- Additional resources, same page (first bullet)

### CODE REVIEW

File: modules/alternatives-to-storing-admin-secrets-in-kube-system.adoc [hunk 1]
```diff
@@ -22,6 +22,16 @@ The Cloud Credential Operator (CCO) manages cloud provider credentials as Kubern
 ifdef::aws,google-cloud-platform[]
 If you prefer not to store an administrator-level credential secret in the cluster `kube-system` project, you can choose one of the following options when installing {product-title}:
 
+endif::aws,google-cloud-platform[]
+
+ifdef::aws[]
+* *Manage cloud credentials manually*:
++
+You can use the CCO utility (`ccoctl`) to configure the CCO to use the Amazon Web Services Security Token Service (AWS STS). When the CCO is configured to use STS, it assigns IAM roles that provide short-term, limited-privilege security credentials to components.
```
Comment (joelddiaz) [inline modules/alternatives-to-storing-admin-secrets-in-kube-system.adoc:30]:
```suggestion
You can use the CCO utility (`ccoctl`) to configure the clustter to use the Amazon Web Services Security Token Service (AWS STS). When the CCU utility is used to configure the cluster for STS, it assigns IAM roles that provide short-term, limited-privilege security credentials to components.
```

**Review by joelddiaz — COMMENTED** [review]
small rewording of things, but otherwise :+1:

**Comment by ahardin-rh** [comment]
Looks good!

**Pull request merged**